microsoft-edge/progressive-web-apps-chromium/whats-new/pwa.md

```diff
@@ -13,6 +13,39 @@ This page lists the updates to PWAs and Web Apps from the Microsoft Edge Web App
 
 
 <!-- ====================================================================== -->
+## What's New in Microsoft Edge 100
+Microsoft Edge Canary/Dev channels moved to version 100 on February 9,2022 and in this milestone, we are testing the following new web app features. 
```

Comment:
Should we expand a bit on what "we are testing" means? What should readers expect to see in 100? Will they get the new features, or not? It might help to set expectations here.